seg_lapa/config/trainer/trainer.yaml

```
@@ -1,12 +1,12 @@
 # @package _group_
 name: trainer
-gpus: "0"
-overfit_batches: 0.0
+gpus: "0,1"
```

```suggestion
gpus: 2
```
Lets keep this an int due to errors in pl with sending gpus ids that do not include gpu0 (I've made this change in my branch, not pushed yet).
See https://github.com/Shreeyak/pytorch-lightning-segmentation-template/pull/15

---

Very weird. The value shown in this diff is different from what is on master..
What I see as master in PR:
```
# @package _group_
name: trainer
gpus: "0"
overfit_batches: 0.0
```

Actual master branch
```

# @package _group_
name: trainer

gpus: 1  # Denotes the number of gpus to use. Set CUDA_VISIBLE_DEVICES env var to control which gpus are used.
overfit_batches: 0.0
```

---

The diff is against the point at which your branch started from master branch, instead of the latest master branch.

In the future, I think we should rebase the branch to master tip before PR, or merge master branch into the PR branch before submitting the PR.

---

I'll merge master into this PR before reviewing further.